superset-frontend/src/cccs-viz/plugins/plugin-chart-at-a-glance-user-id/src/plugin/controlPanel.ts

```diff
@@ -196,6 +196,46 @@ const config: ControlPanelConfig = {
         ],
       ],
     },
+    {
+      label: t('IP dashboard link configuration'),
+      expanded: true,
+      tabOverride: 'data',
+      controlSetRows: [
+        [
+          {
+            name: 'ipDashBoardBaseUrl',
+            config: {
+              type: 'TextControl',
+              label: t('Base url for supersert'),
+              renderTrigger: true,
+              description: t('the url for superset you wish to link to. i.e superset.com'),
```

Comment:
Make this a sentence,
'The URL for superset that you wish to link to (ex. superset.com).'